### PR TITLE
Change pccs server url into environmental variables

### PR DESCRIPTION
**Pull request by panpan0721**
Remove sgx_default_qcnl.conf file generating in Dockfile

Add sgx_config.sh shell script for getting pccs url configration
in environmental and generating sgx_default_qcnl.conf on
docker start.

Add start.sh shell script for calling sgx_config.sh and the docker
work payload command.

Signed-off-by: panpan0721 <2271409777@qq.com>

### CODE REVIEW

File: docker/core/start.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin sh
+sh /home/sgx_config.sh
+node /home/ehsm/ehsm_kms_service/ehsm_kms_server.js
```
Comment (syan10) [inline docker/core/start.sh:3]: combine the twi scripts into one, (named as start_ehsm-kms.sh)

File: docker/dkeycache/start.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin sh
+sh /home/sgx_config.sh
```
Comment (syan10) [inline docker/dkeycache/start.sh:2]: combine and named as "start_dkeycache.sh"

File: docker/dkeyserver/start.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin sh
+sh /home/sgx_config.sh
+sh -c /home/ehsm/out/ehsm-dkeyserver/ehsm-dkeyserver
```
Comment (syan10) [inline docker/dkeyserver/start.sh:3]: combine and named as start_dkeyserver.sh

Comment (panpan0721) [reply docker/dkeyserver/start.sh]: It will be fixed

File: docker/dkeycache/start.sh (hunk 2 above)
Comment (panpan0721) [reply docker/dkeycache/start.sh]: It will be fixed

File: docker/core/start.sh (hunk 1 above)
Comment (panpan0721) [reply docker/core/start.sh]: It will be fixed

**Review by syan10 — APPROVED** [review]

**Pull request merged**